consensus/consensus-types/src/quorum_cert.rs

```diff
@@ -45,23 +45,27 @@ impl QuorumCert {
     }
     /// All the vote data getters are just proxies for retrieving the values from the VoteData
     pub fn certified_block_id(&self) -> HashValue {
-        self.vote_data.block_id()
+        self.vote_data.proposed().id()
     }
 
     pub fn certified_state_id(&self) -> HashValue {
-        self.vote_data.executed_state_id()
+        self.vote_data.proposed().executed_state_id()
     }
 
     pub fn certified_block_round(&self) -> Round {
-        self.vote_data.block_round()
+        self.vote_data.proposed().round()
     }
 
     pub fn parent_block_id(&self) -> HashValue {
-        self.vote_data.parent_block_id()
+        self.vote_data.parent().id()
     }
 
     pub fn parent_block_round(&self) -> Round {
```

Comment:
Should we add `parent_block_state_id()` just for the sake of symmetry? Tbh I'd prefer not to have these wrapper functions at all, but I realize it might be painful to get rid fo these.

Reply:
Let's delete them all but maybe after landing this refactoring so I don't pull in too much?